src/components/Articles.jsx

```diff
@@ -0,0 +1,27 @@
+import ArticleListCard from "./ArticleListCard";
+import { fetchArticles } from "../api";
+import { useState, useEffect } from "react";
+
+const Articles = () => {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    fetchArticles().then((articles) => {
+      setArticles(articles);
+    });
+  }, []);
+
```

Comment:
Could you add some loading state so we get some feedback that the request is happening?